LunarTimes/TabBarViewController.swift

```diff
@@ -45,13 +47,18 @@ protocol LocationChangedDelegate {
 }
```

Comment:
I would change LocationChangedDelegate to `func locationSelected(selectedLocation: SunriseLocation)` as well